packages/eslint-plugin/docs/rules/consistent-type-exports.md

```diff
@@ -13,6 +13,19 @@ This rule aims to standardize the use of type exports style across a codebase.
 
 Given a class `Button`, and an interface `ButtonProps`, examples of code:
 
+
+```ts
+export interface ButtonProps{
+  onClick: () => void;
+}
+
+class Button implements ButtonProps {
+  onClick(){
+    console.log('button!')
+  }
+}
```

Comment:
The example should illustrated re-exporting types. For example:

```suggestion
interface ButtonProps {
  onClick: () => void;
}

class Button implements ButtonProps {
  onClick() {
    console.log('button!');
  }
}

export { Button };
export type { ButtonProps };
```

Probably we should also give an invalid example, like `export { Button, ButtonProps };`

Also, I believe code style can be fixed with Prettier. Please remember to format files before committing.